src/app/components/forms/commons/address_phone_types.js

```diff
@@ -0,0 +1,2 @@
+export const addressTypes = ["Home", "Work", "Other"]
```

Comment:
perhaps we should name this file to something more general like profile definitions or something defines.